storage/innobase/row/row0upd.cc

```diff
@@ -2506,7 +2507,7 @@ row_upd_sec_index_entry(
 		}
 
 #ifdef WITH_WSREP
-		ut_ad(err == DB_SUCCESS || err == DB_LOCK_WAIT
+		ut_ad(err == DB_SUCCESS || err == DB_LOCK_WAIT || err == DB_INTERRUPTED
 		      || err == DB_DEADLOCK || err == DB_LOCK_WAIT_TIMEOUT);
```

Comment:
I do not think that `DB_LOCK_WAIT` is possible here. It should have been resolved to one of the other errors by this point.

Reply:
I see, okay. Let me work on this issue more.